build.gradle

```diff
@@ -208,7 +214,10 @@ static void assertExecutableSpecRequirementsForTests(Project project, Test test)
             'org.concordion.ext.ParallelRunExtension'
 
     test.systemProperties['concordion.output.dir'] = project.reporting.baseDir.getAbsolutePath() + "/spec"
-    test.systemProperties['concordion.run.threadCount'] = '2.5C' // 2.5 * number of cores
+
+    // test all "waiting" classes in parallel so we have less shorter critical path of all long lasting blocking tests
+    // See all BaseNotifiablePayment + ScheduledJob + HttpRequestUtilTimeout fixtures for more details
+    test.systemProperties['concordion.run.threadCount'] = '16'
```

Comment:
why fixed thread count now of 16? I looked at these classes mentioned in the comment, but didn't see any explanation for the thread count.

Reply:
number of `all "waiting" classes` is about 16 now